app/scripts/modules/kubernetes/src/v2/manifest/wizard/BasicSettings.tsx

```diff
@@ -19,7 +19,7 @@ export function ManifestBasicSettings({
   selectedAccount,
 }: IManifestBasicSettingsProps) {
   return (
-    <div className="container-fluid form-horizontal">
```

Comment:
was this class just not doing anything?

Reply:
It was causing some alignment issues if things were added underneath it in other forms.